libtinyaes/test_unit_aes_ccm_s.c

```diff
@@ -1,13 +1,11 @@
-#ifdef WITH_AES_CCM_S
```

Comment:
It would be great if you could add header with original authors and copyright dates of test.

Consider adding yourself with current year. 

Applied to all missing headers.

Reply:
Add also me  (c) 2021, because I'm the original author of eax and ccm which are used in g3-plc stack.

Reply:
Yep it was my intention saying that please add valid authors :)

GCM tests author is @xvuko  (c) 2023